### PR TITLE
show diagnostic when error occurred with --showConfig

### LINKED ISSUE #28598: `--showConfig` is ignoring missing `extends` file
**TypeScript Version:**  3.2.0-dev.20181117

and local build with 608a8307accdfdf0f0ae8fb8b356adedee939d09

**Search Terms:**
showConfig

**Code**

```tsconfig.json
{
  "extends": "./notExists.tsconfig.json",
  "compilerOptions": {
    "target": "esnext",
    "module": "commonjs",
    "strict": true
  }
}
```

**Expected behavior:**
show `error TS5058: The specified path does not exist: '/foobar/notExists.tsconfig.json'.` error message.

**Actual behavior:**
don't report any error and dump incompleteness config value.

**Playground Link:** None

**Related Issues:** #27353

I made a patch to this issue.
If this is a problem, I can submit this patch.
https://github.com/vvakame/TypeScript/commit/d6d9c9ec2053080f4ac632357a422e28e03f7391

### PR DESCRIPTION
**Pull request by vvakame**
Fixes #28598
thanks your comment @weswigham

### CODE REVIEW

File: src/tsc/tsc.ts [hunk 1]
```diff
@@ -133,6 +133,11 @@ namespace ts {
         if (configFileName) {
             const configParseResult = parseConfigFileWithSystem(configFileName, commandLineOptions, sys, reportDiagnostic)!; // TODO: GH#18217
             if (commandLineOptions.showConfig) {
+                if (configParseResult.errors.length !== 0) {
+                    updateReportDiagnostic(configParseResult.options);
+                    configParseResult.errors.forEach(reportDiagnostic);
+                    return sys.exit(ExitStatus.DiagnosticsPresent_OutputsGenerated);
```
Comment (sheetalkamat) [inline src/tsc/tsc.ts]: I think return code should be `DiagnosticsPresent_OutputsSkipped`

Comment (weswigham) [reply src/tsc/tsc.ts]: Seems reasonable.

**Comment by weswigham** [comment]
Btw @DanielRosenwasser I imagine we wanna take this in 3.1 so we ship the feature without immediately changing it in the next release.

File: src/tsc/tsc.ts (hunk 1 above)
Comment (vvakame) [reply src/tsc/tsc.ts]: wow, I forgot to fix it. thanks!

**Comment by DanielRosenwasser** [comment]
Thanks @vvakame!

**Pull request merged**